arithmetization/src/test/java/net/consensys/linea/replaytests/Sepolia9333217.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright ConsenSys Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.replaytests;
+
+import static net.consensys.linea.replaytests.ReplayTestTools.replay;
+import static net.consensys.linea.testing.ReplayExecutionEnvironment.LINEA_SEPOLIA;
+
+import net.consensys.linea.UnitTestWatcher;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@Tag("replay")
+@ExtendWith(UnitTestWatcher.class)
+public class Sepolia9333217 {
```

Comment:
Could you add documentation to this test ? What was the issue ? Is this L2L1 stuff ? EcData ? ...